frontend/packages/kubevirt-plugin/src/utils/validations/vm/disk.ts

```diff
@@ -129,6 +133,11 @@ export const validateDisk = (
       addRequired(pvcName);
       validations.pvc = validatePVCName(pvcName, usedPVCNames);
     }
+
+    if (disk.getType() === DiskType.DISK) {
```

Comment:
I think we can skip this `if`, as all disk types require bus

Reply:
we probably have to deffer this one to a followup if we cannot make CDs support into this PR on time.